lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -342,8 +342,8 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 17, 8),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 17, 8),
+        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 18, 8),
+        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 18, 8),
         # split mongo: 3 queries, regardless of thread response size.
         (ModuleStoreEnum.Type.split, 1, 3, 3, 17, 8),
```

Comment:
I don't think the `bulk_operations` manager should affect django calls directly. Have you tried putting in a breakpoint in where it's making the call, and seeing if it's getting stored during setup (like in the earlier test?)